v3/src/components/graph/components/attribute-label.tsx

```diff
@@ -133,7 +134,7 @@ export const AttributeLabel = observer(
               enter.append('text')
                 .attr('class', className)
                 .attr('text-anchor', anchor)
-                .attr('data-testid', `attribute-label-${place}`)
+                .attr('data-testid', className)
```

Comment:
I have left this to keep using the `className` without the ${place} for now since the tests don't use it and I'd have to include the place context in the method that's using this just to accommodate it, without really needing it.